From: Bastian Germann <bage@linutronix.de>
Date: Mon, 19 Oct 2020 12:24:53 +0200
Subject: d/copyright: Fix public-domain paragraph

The machine-readble copyright format standardizes on the pseudo-license
public-domain for public domain files.  Also, the Copyright field is
missing from the paragraph, so convert the X-Notes to a pseudo-Copyright
notice.
---
 debian/copyright | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/copyright b/debian/copyright
index 096b0d8..989c003 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -10,8 +10,8 @@ License: GPL-2
 Files:
  passwd.master
  group.master
-License: PD
-X-Notes: Originally written by Ian Murdock <imurdock@debian.org> and
+License: public-domain
+Copyright: PD; Originally written by Ian Murdock <imurdock@debian.org> and
  Bruce Perens <bruce@pixar.com>.
 
 Files: doc/*
